### ISSUE TITLE
Reentry v1.1.0 breaks code that used v1.0.3

### ISSUE THREAD

**Issue by sphuber** [DropD/reentry#22]
[The test builds](https://travis-ci.org/aiidateam/aiida_core/jobs/365162956) of `aiida-core` started failing with the following exception:

```
Processing /home/travis/build/aiidateam/aiida_core
    Complete output from command python setup.py egg_info:
    fastentrypoints
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-VOKl_j-build/setup.py", line 126, in <module>
        long_description=open(path.join(aiida_folder, 'README.rst')).read(),
      File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/setuptools/__init__.py", line 129, in setup
        return distutils.core.setup(**attrs)
      File "/opt/python/2.7.14/lib/python2.7/distutils/core.py", line 111, in setup
        _setup_distribution = dist = klass(attrs)
      File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/setuptools/dist.py", line 372, in __init__
        _Distribution.__init__(self, attrs)
      File "/opt/python/2.7.14/lib/python2.7/distutils/dist.py", line 287, in __init__
        self.finalize_options()
      File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/setuptools/dist.py", line 528, in finalize_options
        ep.load()(self, ep.name, value)
      File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2324, in load
        return self.resolve()
      File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2330, in resolve
        module = __import__(self.module_name, fromlist=['__name__'], level=0)
      File "/tmp/pip-VOKl_j-build/.eggs/reentry-1.1.0-py2.7.egg/reentry/__init__.py", line 2, in <module>
        from reentry.manager import DEFAULT_MANAGER as manager
      File "/tmp/pip-VOKl_j-build/.eggs/reentry-1.1.0-py2.7.egg/reentry/manager.py", line 16, in <module>
        class PluginManager(object):
      File "/tmp/pip-VOKl_j-build/.eggs/reentry-1.1.0-py2.7.egg/reentry/manager.py", line 19, in PluginManager
        def __init__(self, backend=get_default_backend()):
      File "/tmp/pip-VOKl_j-build/.eggs/reentry-1.1.0-py2.7.egg/reentry/manager.py", line 12, in get_default_backend
        from reentry.config import get_datafile
      File "/tmp/pip-VOKl_j-build/.eggs/reentry-1.1.0-py2.7.egg/reentry/config.py", line 3, in <module>
        from configparser import ConfigParser
    ImportError: No module named configparser

**Comment by giovannipizzi** [comment]
This is quite important to fix soon: with this bug no people is able to install anymore AiiDA

**Comment by DropD** [comment]
fixed in v1.1.1
